### PR TITLE
Add Yunohost to the list of possible installations

### PR DESCRIPTION
**Pull request by Thovi98**
Hello 👋

Congrats for your work here!

Install is quite easy and I’ve managed to make a package for [Yunohost](https://yunohost.org/) (a debian based OS that aims to simplifying self-hosting).

Feel free to merge this if you find it useful.

We have an "automatic updater" which makes a PR to the package once a day if a commit has been made upstream, so the package stays relatively well updated.

### CODE REVIEW

**Comment by kberg** [comment]
Thank you for this! I've never heard of Yunohost, and peeking at the link
you sent, it doesn't show what it's actually installing. Is there a way to
provide more open information than the automatic updater?

**Comment by Thovi98** [comment]
Okay sorry I wasn’t clear.

This is the package repo to install TM in Yunohost:
https://github.com/YunoHost-Apps/terraforming-mars_ynh/

The app can be found in the Yunohost app’s list: https://apps.yunohost.org/app/terraforming-mars

Is it what you are looking for? If you have other questions I can try to answer them :)

**Review by kberg — APPROVED** [review]
I made a couple of minor adjustments but this is good. Thank you!

File: README.md [hunk 1]
```diff
@@ -58,6 +58,15 @@ as they're the clearest instructions.)
 ## ⬤ I want to run a copy of the server on Docker
 Check out our [Docker setup wiki page](https://github.com/bafolts/terraforming-mars/wiki/Docker-Setup)
 
+(Warning, this is not aggressively supported, though some people are on the Discord.)
+
+## ⬤ I want to run a copy on a YunoHost server
+[![Install Terraforming Mars with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app=terraforming-mars)
+
+The code describing the Yunohost is in this [GitHub repo](https://github.com/YunoHost-Apps/terraforming-mars_ynh)
```
Comment (Thovi98) [inline README.md:66]:
```suggestion
The code for the Yunohost Terraforming-Mars package is in this [GitHub repo](https://github.com/YunoHost-Apps/terraforming-mars_ynh)
```
Yunohost is the OS, this is only a package of the app for Yunohost :)

File: README.md [hunk 2]
```diff
@@ -58,6 +58,15 @@ as they're the clearest instructions.)
...
+
+(Warning, this is not specifically supported.)
```
Comment (Thovi98) [inline README.md:68]:
```suggestion
(Warning, this is not specifically supported. If you have an issue or a question, feel free to create an issue [here](https://github.com/YunoHost-Apps/terraforming-mars_ynh)
```

File: README.md (hunk 1 above)
Comment (kberg) [reply README.md:66]: Mea culpa.

**Comment by kberg** [comment]
Thank you!

**Pull request merged**